pkg/registry/apis/provisioning/usage.go

```diff
@@ -59,6 +59,10 @@ func (b *APIBuilder) collectProvisioningStats(ctx context.Context) (metrics map[
 	}
 
 	// Inspect all configs
+	if b.repositoryLister == nil {
+		span.SetStatus(codes.Ok, "repository lister is not available yet")
+		return m, nil
```

Comment:
`return nil, errors.New("repository lister should be available but it is not")`

(and no span status set)

Reply:
Fixed here: https://github.com/grafana/grafana/pull/108173